app/webserver/webserver.go

```diff
@@ -134,28 +131,37 @@ func wsReader(conn *websocket.Conn) {
 			createChatMessage := CreateChatMessage{}
 			json.Unmarshal([]byte(p), &createChatMessage)
 			log.Println("[axolotl] Create chat for ", createChatMessage.Tel)
-			createChat(createChatMessage.Tel)
-			activeChat = createChatMessage.Tel
+			newChat := createChat(createChatMessage.Tel)
+			activeChat = newChat.ID
 			store.ActiveSessionID = activeChat
-			s := store.SessionsModel.Get(createChatMessage.Tel)
+			s := store.SessionsModel.GetByE164(createChatMessage.Tel)
 			sendCurrentChat(s)
 		case "openChat":
 			openChatMessage := OpenChatMessage{}
 			json.Unmarshal([]byte(p), &openChatMessage)
-			s := store.SessionsModel.Get(openChatMessage.Id)
-			log.Println("[axolotl] Open chat with id: ", s.ID)
-			activeChat = openChatMessage.Id
-			store.ActiveSessionID = activeChat
-			sendCurrentChat(s)
+
+			log.Println("[axolotl] Open chat with id: ", openChatMessage.Id)
+			s, err := store.SessionsModel.Get(openChatMessage.Id)
+			if err != nil {
+				log.Errorln("[axolotl] Open chat with id: ", openChatMessage.Id, "failed", err)
+			} else {
+				activeChat = openChatMessage.Id
+				if !s.IsGroup {
+					// p, _ := textsecure.GetProfile(s.Tel)
```

Comment:
Work to be done there? If so, write a detailed TODO please ;)

Reply:
Done